lib/renderer.js

```diff
@@ -1,21 +1,28 @@
 'use strict';
 
 const less = require('less');
-const { basename, dirname, join} = require('path');
+const { basename, dirname, join } = require('path');
+const micromatch = require('micromatch');
 
 module.exports = async function(data) {
   this.theme.config.less = Object.assign({
     paths: []
   }, this.theme.config.less);
+  const { route, theme } = this;
 
-  const config = this.theme.config.less;
+  const config = theme.config.less;
   const cwd = process.cwd();
-  const paths = config.paths.map(filepath => {
-    return join(cwd, filepath); // assuming paths are relative from the root of the project
-  });
+  let { paths } = config;
+  const routeList = route.list();
```

Comment:
Probably not the correct approach, as `paths:` usually refers to a folder outside of hexo processing (e.g. node_modules), which is not intended to copied into 'public/' folder.